go-controller/pkg/ovn/ovn.go

```diff
@@ -180,6 +182,16 @@ func (oc *DefaultNetworkController) ensureLocalZonePod(oldPod, pod *kapi.Pod, ad
 		}
 	}
 
+	if kubevirt.IsPodLiveMigratable(pod) {
+		switchNames, err := oc.getSwitchNames(pod)
+		if err != nil {
+			return fmt.Errorf("failed retrieving switch names when ensuring local zone for pod %s/%s: %v", pod.Namespace, pod.Name, err)
+		}
+		if err := kubevirt.EnsureRoutingForVM(oc.controllerName, oc.watchFactory, oc.nbClient, switchNames, true /* local zone */, pod, oc.GetNetworkName()); err != nil {
```

Comment:
Should we consider doing this before `addLogicalPort` to provide the Pod the maximum connectivity possible before the CNI attaches the pod?

Reply:
As we commented, if we do so we don't have the the ip address at target pod to do the point to point routing.